schema2proto-lib/src/test/resources/xsd/basic/unboundednestedsequences.xsd

```diff
@@ -15,4 +15,25 @@
             </xsd:element>
         </xsd:sequence>
     </xsd:complexType>
+
+    <xsd:complexType name="Outer2">
+        <xsd:sequence>
+            <xsd:element name="a">
+                <xsd:complexType>
+                    <xsd:sequence>
+                        <xsd:element name="b" type="xsd:string" />
+                        <xsd:sequence minOccurs="0" maxOccurs="unbounded">
+                            <xsd:element name="c" type="xsd:string" />
+                            <xsd:sequence maxOccurs="unbounded">
```

Comment:
Make this minOccurs=1 to test that correct use of repeat is generated

Reply:
minOccurs=1 is the default value when omitted.